broker/src/main/java/io/zeebe/broker/system/SystemContext.java

```diff
@@ -29,6 +31,8 @@
       "Replication factor %s needs to be larger then zero and not larger then cluster size %s.";
   private static final String SNAPSHOT_PERIOD_ERROR_MSG =
       "Snapshot period %s needs to be larger then or equals to one minute.";
+  private static final String MMAP_REPLICATION_ERROR_MSG =
+      "Using memory mapped storage level is currently unsafe with replication enabled; if you wish to use replication, set useMmap to false";
```

Comment:
Maxybe a little more context would be helpful in the how to fix part:
E.g. set BrokertCfg->DataCfg-useMmap to false

Reply:
Will update with env var to look for/update :+1: